CONTRIBUTING.md

```diff
@@ -50,5 +50,14 @@ For full coverage, please run unit tests against debug and test for both x86 and
 `runtests.cmd` looks for the build output in the default build output folder `Build\VcBuild\bin`. If the build output path is changed from this default then use the `-bindir` switch to specify that path.
 
 ### Code Flow into Microsoft Edge
+
 Changes that make it into our ChakraCore GitHub master branch have a short journey to Chakra.dll. Code flows daily from GitHub to the internal repository from which builds of Chakra.dll are produced and then it flows into Windows and Microsoft Edge. While code flows quickly on this first leg of the journey, code flow from our internal branch to a Windows flighting branch is subject to any number of delays. So it is difficult to predict when your change in our GitHub repo will make it into a particular Windows flight.
-#
+
+### Issue Labels
+
+ - [`Accepting PRs`](https://github.com/Microsoft/ChakraCore/labels/Accepting%20PRs): these issues are specifically well suited for outside contributors.
+ - [`Your first PR`](https://github.com/Microsoft/ChakraCore/labels/Your%20first%20PR): these issues are well suited for people new to GitHub contributions -- we're here to help you get started in OSS.  These issues are typically limited in scope and impact and thus are good starting points to become familiar with being a contributor to ChakraCore.
```

Comment:
IMO both new github contributors and new ChakraCore contributors can benefit from it. Even github veterans can use it as a chance to familiarize themselves with our contributor guideline, code of conduct, codebase & running tests. These's no harm from extending the label's target a little bit; it's really just an indicator of easier tasks.